src/context/WalletProvider/Ledger/components/Connect.tsx

```diff
@@ -57,8 +65,26 @@ export const LedgerConnect = ({ history }: LedgerSetupProps) => {
         // Remove all provider event listeners from previously connected wallets
         await removeAccountsAndChainListeners()
 
+        // Pair the device, which gets approval from the browser to communicate with the Ledger USB device
         const wallet = await adapter.pairDevice()
 
+        // Check the number of connected devices. Needs a device to be paired for this to work (see above)
+        const numDevices = await handleCheckNumDevices()
```

Comment:
q: Is the intent to have this displayed *before* "Pair new/existing device" click, like in these two recordings?

> At least 1 device check:

https://github.com/shapeshift/web/assets/125113430/8b51ac2c-d5bf-4bb6-a9da-beef638f9472

> At most 1 device check:

https://github.com/shapeshift/web/assets/125113430/915c4b05-7e5b-4ba6-ac38-c21675ceb829

Reply:
Correct yeah, the idea is that you cannot pair any ledger device until exactly one is connected

Reply:
Correction to my above reply (this PR was authored a long time ago), the flow has changed as the browser needs to pair webusb before requesting devices. This was picked up during ops testing where the browser needs an existing pairing with a device before we can make requests to determine the number of connected devices.

I'll upload updated screen recordings.